### PR TITLE
Release/23.2.2

### PR DESCRIPTION
**Pull request by skyflow-bharti**

### CODE REVIEW

File: tests/core/internal/skyflow-frame/skyflow-frame-controller.test.js [hunk 1]
```diff
@@ -468,6 +474,31 @@ describe('Retrieving data using get', () => {
       done();
     }, 1000);
   });
+  test('get success2', (done) => {
```
Comment (Shaik-Luqmaan) [inline tests/core/internal/skyflow-frame/skyflow-frame-controller.test.js:477]: Pls add proper message for unit tests

**Review by Shaik-Luqmaan — CHANGES_REQUESTED** [review]

File: src/core-utils/collect.ts [hunk 2]
```diff
@@ -141,9 +183,97 @@ export const constructElementsInsertReq = (req, options) => {
...
+  const table = skyflowIdRecord.fields.table;
+  const skyflowID = skyflowIdRecord.skyflowID;
+  skyflowIdRecord.fields = _.omit(skyflowIdRecord.fields, 'table');
+  skyflowIdRecord.fields = _.omit(skyflowIdRecord.fields, 'skyflowID');
+  return client.request({
+    body: {
+      record: {
+        fields: { ...skyflowIdRecord.fields },
+      },
+      tokenization: options.tokens,
```
[the remark this answers is not in the document]
Comment (skyflow-bharti) [reply src/core-utils/collect.ts]: Added

File: tests/core/internal/skyflow-frame/skyflow-frame-controller.test.js (hunk 1 above)
Comment (skyflow-bharti) [reply tests/core/internal/skyflow-frame/skyflow-frame-controller.test.js]: Added proper messsage for this test case

**Comment by skyflow-bharti** [comment]
codecov/patch uncovered lines will be tested in integration tests

**Pull request merged**